bin/metrics-mesos.rb

```diff
@@ -33,17 +33,7 @@
 require 'socket'
 require 'json'
 
-# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
-MASTER_DEFAULT_PORT = '5050'.freeze
-SLAVE_DEFAULT_PORT = '5051'.freeze
-
 class MesosMetrics < Sensu::Plugin::Metric::CLI::Graphite
-  option :mode,
```

Comment:
I think that we should leave mode so that you pass port and it can be independent, I am about to make a similar comment below on if port == foo which is super brittle.